packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx

```diff
@@ -122,10 +128,16 @@ const PatientListDetailComponent = () => {
               </>
             }>
             <OverflowMenuItem
-              itemText={t('editNameDescription', 'Edit Name/ Description')}
+              className={styles.menuItem}
+              itemText={t('editNameDescription', 'Edit Name or Description')}
               onClick={() => setEditPatientListDetailOverlay(true)}
             />
-            <OverflowMenuItem itemText={t('delete', 'Delete')} onClick={handleDelete} isDelete />
+            <OverflowMenuItem
+              className={styles.menuItem}
+              itemText={t('deletePatientList', 'Delete patient list')}
```

Comment:
I don't see this string in the translation files. Make sure you run the git commit hooks.

Reply:
@jwnasambu fixing Ian's comment above will help fix the failing test.